plane/src/main.rs

```diff
@@ -79,6 +81,17 @@ enum Command {
         /// Optional pool identifier. If present, will only schedule workloads with a matching `pool` tag on this drone.
         #[clap(long)]
         pool: Option<String>,
+
+        /// Automatically prune stopped images.
+        /// This prunes *all* unused container images, not just ones that Plane has loaded, so it is disabled by default.
+        #[clap(long)]
+        auto_prune: bool,
+
+        /// Minimum age (in seconds) of backends to prune.
+        /// By default, all stopped backends are pruned, but you can set this to a positive number of seconds to prune
+        /// only backends that were created more than this many seconds ago.
+        #[clap(long, default_value = "0")]
+        cleanup_min_age_seconds: i32,
```

Comment:
I think it's easy to get confused over what these two flags are meant to do. `auto_prune` applies only to pruning images and `cleanup_min_age_seconds` applies only to pruning containers. I wonder if there's a different name we could give these (maybe `auto_prune_images` and/or `prune_containers_min_age_secs`). I think updating the comments to emphasize that one is for images and the other for containers would also help.